db/migrate/20171121165127_create_u2f_registrations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateU2fRegistrations < ActiveRecord::Migration[5.0]
+  def change
+    create_table "u2f_registrations" do |t|
```

Comment:
Should this use uuids for the id?

```ruby
create_table "u2f_registrations", id: :uuid do |t|
```

Reply:
@lgebhardt What is the motivation for using a UUID here over an integer?

General Brave best practice, or a technical issue you see... etc? We could use a UUID, I know the publisher's table does, but generally I default to an integer.

Reply:
This is a good catch @lgebhardt - we had just noticed yesterday that this app has been using UUID IDs for many models. I would agree that we may want to use UUIDs here as well.

Reply:
@mixonic I was thinking consistency. I'm not sure there's a compelling technical reason in this case.

Reply:
@mixonic I would assume that actions like Delete may reference a URL containing the ID, and using a UUID here would avoid revealing incremented ID's in those URLs. I would see that as an advantage.

Reply:
We went ahead and made this change. Apparently you can set the default primary key type (per https://github.com/rails/rails/issues/23422#issuecomment-326596492) however there is no way to set the default foreign key type which is something that tripped us up when we created the relationship to the publishers model. Seems tricky to navigate in the future, but there it is.

Thanks for the 👀  @lgebhardt